code.js

```diff
@@ -698,6 +699,12 @@ function toggleTheme() {
   );
 }
 
+function onAnchorClick() {
+  if (location.hash.length !== 0) {
+    window.scrollTo(window.scrollX, window.scrollY - 100);
```

Comment:
What does the 100 represent? Just for readability can you create a variable and assign it the value 100 so it can tell us what it means. For example:
`window.scrollTo(window.scrollX, window.scrollY - marginHeight);`

Reply:
Thanks, agreed, it looks like magic number. Changed.